R/Incremental.R

```diff
@@ -187,7 +187,7 @@ saveIncremental <- function(data, fileName, ...) {
     if (length(idx) > 0) {
       previousData <- previousData[-idx, ]
     }
-    data <- dplyr::bind_rows(previousData, data)
```

Comment:
@ablack3  isnt the output of rbind not tibble? if yes, i think the loss of type stability may cause other problems

Reply:
Yea I'm removing the strictness imposed by dplyr::bind_rows. I don't think my fix is actually a good idea :)
It needs more investigation to find a better fix. But what I did at least allowed the code to run for the SOS study.